.chloggen/deprecate-NewUnhealthyProcessorCreateSettings.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: processortest
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Deprecate and remove processortest.NewUnhealthyProcessorCreateSettings
```

```suggestion
note: Deprecate processortest.NewUnhealthyProcessorCreateSettings
```

this isn't removed yet